contact.html

```diff
@@ -58,30 +58,30 @@
               </button>
 
               <!-- Logo -->
-              <a class="navbar-brand" href="../index.html"><img class="navbar-logo" src="template_assets/assets/images/logo.png" alt="Definity - Logo"></a>
+              <a class="navbar-brand" href="./index.html" style="font-size: 24px">W.G. Pappas, D.D.S.</a>
```

Comment:
The title text needs to be in the true vertical center of the navbar

<img width="555" alt="Screen Shot 2019-10-04 at 3 19 08 PM" src="https://user-images.githubusercontent.com/10214737/66237308-67208200-e6ba-11e9-9c28-aea7ec8cf262.png">

Reply:
The styling for the navbar needs to be consistent on all of the pages, and you can put the styling in the CSS so it applies to all pages easily